kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala

```diff
@@ -168,12 +168,16 @@ private[kyuubi] object Utils extends Logging {
     System.getProperty(IS_TESTING.key) != null
   }
 
+  val DEFAULT_SHUTDOWN_PRIORITY = 100
+  val SERVER_SHUTDOWN_PRIORITY = 75
+  val SPARK_CONTEXT_SHUTDOWN_PRIORITY = 50
```

Comment:
We can add some comment that this priority depends on `org.apache.spark.util.ShutdownHookManager.SPARK_CONTEXT_SHUTDOWN_PRIORITY`

Reply:
a small or big one has a higher priority?

Reply:
big